src/debugProtocol/events/requests/VariablesRequest.ts

```diff
@@ -109,6 +135,16 @@ export class VariablesRequest implements ProtocolRequest {
          */
         enableForceCaseInsensitivity: undefined as boolean,
 
+        /**
+         * Indicates whether the VARIABLES response should include virtual keys for the requested path
+         */
+        getVirtualKeys: undefined as boolean,
+
+        /**
+         * Enables the client application to send path_is_virtual data
+         */
+        includesVirtualPath: undefined as boolean,
+
```

Comment:
I don't think we need this...we derive it from the variable path entries.

Reply:
we need to send and read this.